bundles/org.openhab.ui/web/src/components/pagedesigner/sitemap/widget-details.vue

```diff
@@ -67,42 +76,56 @@
   </f7-card>
 </template>
 
-<style>
-
+<style lang="stylus">
+.widget-detail
+  .item-inner:after
+    height 0px !important /* remove all lines between params */
```

Comment:
This is better codestyle, I will commit that myself.

```suggestion
    height 0 !important /* remove all lines between params */
```